blob/driver/driver.go

```diff
@@ -103,6 +103,44 @@ type Attributes struct {
 	AsFunc func(interface{}) bool
 }
 
+// ListOptions sets options for listing objects in the bucket.
+// TODO(Issue #541): Add Delimiter.
+type ListOptions struct {
+	// Prefix indicates that only results with the given prefix should be
+	// returned.
+	Prefix string
+
+	// PageSize sets the maximum number of objects that will be returned in
+	// a single call. It is guaranteed to be > 0 and <= blob.MaxPageSize.
+	PageSize int
+	// PageToken may be filled in with the NextPageToken from a previous
+	// ListPaged call.
+	PageToken []byte
+}
+
+// ListObject represents a specific blob object returned from ListPaged.
+type ListObject struct {
+	// Key is the key for this blob.
+	Key string
+	// ModTime is the time the blob object was last modified.
```

Comment:
Optional: something about being the zero time if not available?

Reply:
No needed; it's always available (for now, at least).